### PR TITLE
Document the Activity app

### PR DESCRIPTION
**Pull request by settermjd**
This relates to #257.

### CODE REVIEW

**Comment by mmattel** [comment]
You need to add the `modules/user_manual/pages/apps/activity.adoc` page into the navigation.
At least I have not found it there...

**Comment by settermjd** [comment]
> You need to add the `modules/user_manual/pages/apps/activity.adoc` page into the navigation.
> At least I have not found it there...

Thanks. Will add that.

**Comment by settermjd** [comment]
> You need to add the `modules/user_manual/pages/apps/activity.adoc` page into the navigation.
> At least I have not found it there...

Done.

File: modules/admin_manual/pages/configuration/server/activity_configuration.adoc [hunk 1]
```diff
@@ -21,7 +21,11 @@ If it is not enabled, go to your ownCloud Apps page to enable it.
 
 == Configuring your ownCloud for the Activity App
 
+[TIP] 
+====
 A working e-mail configuration is required to configure your ownCloud to send out e-mail notifications.
 Furthermore, it is recommended to configure the xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[background job] `Webcron` or `Cron`.
+====
 
+Email notifications for shared files can be enabled/disabled by administrators with the "_Allow users to send mail notifications for shared files to other users_", available in menu:Settings[Admin > Sharing].
```
Comment (pmaier1) [inline modules/admin_manual/pages/configuration/server/activity_configuration.adoc]: This has nothing to do with the Activity app. The setting is to allow/disallow sending public links via mail from within the web interface.

**Pull request merged**